Refactor solution for Metrics ++ Polish properties

Sgitario opened the pull request:
The changes in this PR are:
- Change the tracing/metrics format to follow the same format as the rest of the properties

This change allows to enable/disable/configure these extensions per module.

- Refactor the metrics module to decouple the metrics of prometheus and have another type of exporter: file.

My idea is to integrate the metrics within the running Quarkus application, so we can assert metrics during the test execution without needing to have a Prometheus.

Also, we're exporting the metrics into a file (/target/logs/metrics.out).

pjgg commented on misc/Metrics.md:
```diff
@@ -1,36 +1,34 @@
+   - `tests_success` represent the total amount of succeed test
+   - `tests_ignored` represent the total amount of ignored test
+   - `tests_fail` represent the total amount of failures 
+   - `module_name` represents the maven modules name
```
Currently, the module name is the project root folder. The problem is that, If we have tree scenarios we will have tree moduleNames with the same name, and that is not very useful. What about replace `module_name` by `scenario_name` where scenario name is `rootFolderName_testClassName`

Sgitario replied:
+1

Sgitario replied:
I kept the `module_name` and added a new one only for the scenario name `scenario_name` (for `testClassName`). 
Let me know if it looks better now.

pjgg commented on misc/Metrics.md:
```diff
@@ -1,36 +1,34 @@
-        - `ts_quarkus_module_status` could be `module_success` or `module_fail`
-   - `ts_quarkus_histogram_modules_duration`
+   - `tests_total` represent the total amount of test
+   - `tests_success` represent the total amount of succeed test
```
what about... test_successed?

Sgitario replied:
+1

pjgg commented on misc/Metrics.md:
```diff
@@ -1,36 +1,34 @@
-        - `ts_quarkus_module_name` represents the maven modules name
-        - `ts_quarkus_module_status` could be `module_success` or `module_fail`
-   - `ts_quarkus_histogram_modules_duration`
+   - `tests_total` represent the total amount of test
```
tests_amount?

Sgitario replied:
I prefer either `total` or `count`, wdyt?

pjgg commented on misc/Metrics.md:
```diff
@@ -1,36 +1,34 @@
+   - `tests_total` represent the total amount of test
+   - `tests_success` represent the total amount of succeed test
+   - `tests_ignored` represent the total amount of ignored test
+   - `tests_fail` represent the total amount of failures 
```
tests_failed?

Sgitario replied:
+1

Sgitario commented:
@pjgg I've already addressed all your comments:
- `module_name` remains the same, but I've added an additional label `scenario_name` with the running class name
- renamed `module_duration_seconds` to `scenario_duration_seconds`
- renamed all the metrics from `success` to `succeed` and `fail` to `failed`
